decidim-proposals/spec/lib/decidim/proposals/import/proposal_creator_spec.rb

```diff
@@ -49,7 +49,6 @@
 
   describe "#resource_attributes" do
     it "returns the attributes hash" do
-      # rubocop:disable Style/HashSyntax
```

Comment:
Could you also remove the re-enabling of the rule below please?

Cannot add a suggestion because the line is too far from this change.

Reply:
actually, this will be removed in the next PR i am working ... https://github.com/tremend-cofe/decidim/pull/233/files#diff-4b9cc73e58932051f5ce5c7a2787606841f06c28edc158456184959b399d6161L63

Reply:
fixed in this PR in [de5b3aa](https://github.com/decidim/decidim/pull/10924/commits/de5b3aa7eeafb2a203198f0d5c1b19f5b3477fb0)